File: SimActivationBackEnd/UserMS/src/main/java/com/ab/sim/user/entity/Identification.java
package com.ab.sim.user.entity;

import lombok.*;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
@Table(name = "identification")
public class Identification {
    @Id
    @Column(name = "id_number")
    String idNumber;
    @Column(name = "id_type")
    String idType;
    String state;
    @Column(name = "first_name")
    String firstName;
    @Column(name = "last_name")
    String lastName;
    String dob;
}

File: SimActivationFrontEnd/src/app/models/identification.ts
export class IdentificationModel {
    idNumber: string;
    idType: string;
    state: string;
    firstName: string;
    lastName: string;
    dateOfBirth: string;

    constructor(id: string, type: string, state: string, fname: string, lname: string, dob: string) {
        this.idNumber = id;
        this.idType = type;
        this.state = state;
        this.firstName = fname;
        this.lastName = lname;
        this.dateOfBirth = dob
    }
}